### DISCUSSION TITLE
MTGの議題は事前にDiscussionsで意見を出し合いたい

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#145] [【運営】相談・報告]
MTGで話し合う内容を
事前にDiscussionsで議題にあげて、個々の意見を確認して
ふわっとしたいたり、空気感を知りたい、話して詳細をつめたいというものをMTGの議題にあげるのが良いのかなと思いました。

イメージとしては、
毎週金曜までに話したい内容をDiscussionで投稿しておいて、土曜日曜で文章で情報共有し、日曜日に最終確認・まとめ上げる感じです。

意見がはっきりまとまっていないものがあれば、それは来週に回したり
MTG中でわいた新たなトピックは来週へ...など。
急ぎのもの、重要なものがあればメンション飛ばしたり、週報用のDiscussionsにDiscussionの番号をはったり...！

どう思いますか？

個人的にMTGの時間が1時間ぐらいに短縮できたら嬉しいな...！と少し思っています🥺🥺🥺

**Comment by kazzyfrog** [comment]
良いと思います！
それでやってみましょう！

**Reply by kazzyfrog** [reply]
第３回のMTGの記録のように、
トピック追加した人が、トップの投稿に目次に追加して、
チャット上で決まったか、MTGで議題に上げるか、をみえると良いかも？ですね！

決定したことはさらっと確認するだけで、決定しなかったことを話し合う的な！

**Reply by pss-aileen** [reply]
そうです！！
第4回のみました！目次そんな感じでできたら良いと思いました！

<img src="https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/140990100/e31df88e-a69d-4222-aad0-b810e5aeb504" width="500">

**Reply by kagomen** [reply]
良いと思います！✨
目次をトピック立てた人が書くシステムも、賛成です！

**Reply by kagomen** [reply]
あと、一週間やったことの報告は無くしても良いかも？と思いましたが、どうでしょうか❓

**Reply by kazzyfrog** [reply]
確かにそうですね！時間的に削って良いかもです！👍

**Comment by pss-aileen** [comment]
あまり話がまとまっていない、調査がより必要なものは、MTGでの会議は見送くる。
次の週までの宿題？とするのが良いかも。
